files/build_templates/docker_image_ctl.j2

```diff
@@ -228,6 +244,10 @@ start() {
             done
         fi
         {%- endif %}
+        # single NPU systems and container running on the host network continue to use the json-file as log-driver
```

Comment:
># single NPU systems and container running on the host network continue to use the json-file as log-driver [](start = 8, length = 106)

The different behavior of single NPU vs multiple NPU is confusing and complex deign.

What about moving all use cases to log-driver, or all use cases to json-file as before?

Reply:
Changed to use json-file for all cases